behat.yml

```diff
@@ -0,0 +1,88 @@
+default:
+  autoload: [ '%paths.base%/tests/behat/bootstrap' ]
+  gherkin:
+    # Disable caching during development. It is enabled for profiles below.
+    cache: ~
+    filters:
+      tags: "~@suggest&&~@skipped"
+  suites:
+    default:
+      paths: 
+        - '%paths.base%/tests/behat/features'
+        - '%paths.base%/modules/tide_event/tests/behat/features'
+        - '%paths.base%/modules/tide_news/tests/behat/features'
+        - '%paths.base%/modules/tide_grant/tests/behat/features'
+        - '%paths.base%/modules/tide_site/tests/behat/features'
+        - '%paths.base%/modules/tide_landing_page/tests/behat/features'
+        - '%paths.base%/modules/tide_api/tests/behat/features'
+        - '%paths.base%/modules/tide_media/tests/behat/features'
+        - '%paths.base%/modules/tide_webform/tests/behat/features'
+        - '%paths.base%/modules/tide_publication/tests/behat/features'
+        - '%paths.base%/modules/tide_alert/tests/behat/features'
+      contexts:
+      - Tide\Tests\Context\FeatureContext
```

Comment:
`FeatureContext` will be shared across all modules, so it needs to be refactored into a namespaced format.